README.md

```diff
@@ -2,3 +2,16 @@
 A repository to replicate registers data from master db to presentation app databases
 
 [![Build Status](https://travis-ci.org/openregister/indexer.svg?branch=master)](https://travis-ci.org/openregister/indexer)
+
+# Requirements
+
+- Java 1.8+
+- Postgres DB 9.4
```

Comment:
nit: I'm on postgres 9.5.1 and it seems to work. maybe `>= 9.4` or `9.4+` as above?